myapp/db/migrate/20230314050752_add_column_is_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnIsAdminToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :is_admin, :boolean, null: false, default: false, after: :password_digest
```

Comment:
I assume just two role,  `normal` and `admin` for this application.

Reply:
In Ruby, it is customary not to use `is` or `has` for variables or methods. So simply `admin` is a good choice.

Reply:
@zackey-y 
Yeah that't right. I fixed that. https://github.com/Fablic/training/commit/a9e7a0f2a99551e7c6f3eef0aceefa879c49a500